MAUI/Cards/getting-started.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: Getting started with MAUI Card control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Card (SfCalendar) control and its basic features.
```

Comment:
description: Learn here about getting started with Syncfusion .NET MAUI Card (SfCalendar) control and its basic features.